app/src/main/res/drawable/ic_android_white_24dp.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="@color/accent_black"
```

Comment:
We used this for the drawer, won't the drawer have a green android icon?

Reply:
I thought the same thing, but it seems the original color is ignored and replaced with a theme color, `?attr/colorPrimary` when active else `?attr/colorOnSurface`.
Although I am still not sure why APK files icon kept their color, maybe because the context object doesn't have a theme associated with it?
https://material.io/components/navigation-drawer/android#anatomy